contributing/getting-started-e2e.md

```diff
@@ -189,7 +189,7 @@ amphtml uses Node.js, the Yarn package manager and the Gulp build system to buil
    You can install Gulp using Yarn:
 
    ```
-   yarn global add gulp
+   yarn global add gulp-cli
```

Comment:
I advocate for using `npx` instead. Global installed packages are as sin.

Reply:
You still have to install `npx` somehow, and that project recommends doing so with `npm install -g`, so it winds up with the same number of global installs while also having to change all the guidance everywhere else from `gulp` to `npx gulp`. But I can do that if you want.

Reply:
`npx` comes bundled with node, just like `npm`.